README.md

```diff
@@ -18,7 +18,7 @@ npm install @libsql/client
 
 ### Importing
```

Comment:
We can also describe how to import the package in Deno. In a Netlify Edge function, the following import works:
```
import { createClient } from "https://esm.sh/@libsql/client@[version]/web"; // replace [version] with the client version
```

And in vanilla Deno:
```
import { createClient } from "npm:@libsql/client@[version]/web"; // replace [version] with the client version
```

The installation section may also mention that in Deno environments, including Netlify Edge functions, there is no installation step.

Reply:
Done